src/components/Card/Card.tsx

```diff
@@ -13,7 +13,12 @@ export type CardProps = {
   radius?: SpacingKey;
 };
 
-const Card: React.FCC<CardProps> = ({ elevated, isTransparent = false, radius, children }) => {
+const Card: React.FC<React.PropsWithChildren<CardProps>> = ({
```

Comment:
this could probably be FCC

Reply:
changed here https://github.com/Orfium/orfium-ictinus/pull/722/commits/47a8ac4868aa9ef425efe5b4747e9db9d7656fc2 good catch